src/dune/action_ast.ml

```diff
@@ -185,6 +185,11 @@ struct
           ; ( "no-infer"
             , Dune_lang.Syntax.since Stanza.syntax (2, 6) >>> t >>| fun t ->
               No_infer t )
+          ; ( "format-dune-file"
+            , Dune_lang.Syntax.since Stanza.syntax (2, 7) >>>
```

Comment:
I versioned the action, even though it is not supposed to be used by users.

Reply:
In fact, we don't need to parse it. We do that for a few other private actions.

Reply:
OK, I took this out.